### ISSUE TITLE
Log frequently unavailable running in Docker container on Synology

### ISSUE THREAD

**Issue by HonestJohn61** [dchesterton/texecom2mqtt-hassio#81]
**Describe the bug**
Using the Synology GUI, the log page frequently does not populate and after a period the Synology GUI reports that the action cannot be carried out becasue the system is busy. Even though the log page does not populate, other functions remain operational. Restarting the Docker container clears the problem and the log will then appear in the GUI

**Application version**
1.2.3

**Texecom alarm type**
Premier Elite 48

**Home Assistant version**
The version of Home Assistant, if applicable.

**Debug log**
Include the app log output to help diagnose the issue. If possible, please add `log: debug` to your config to get the debug log with more information.

**Comment by dchesterton** [comment]
I don't think this is an issue with the app as it just logs the same as any container. Closing this.
